blueocean-dashboard/src/main/js/components/karaoke/components/Step.jsx

```diff
@@ -117,26 +122,27 @@ export class Step extends Component {
         if (logArray && !step.isInputStep) {
             const currentLogUrl = prefixIfNeeded(this.pager.currentLogUrl);
             logger.debug('Updating children');
-            children = (<LogConsole {...{
-                t,
-                router,
-                location,
-                hasMore,
-                scrollToBottom,
-                logArray,
-                currentLogUrl,
-                key: step.logUrl,
-                prefix: `step-${step.id}-`,
-            }}
-            />);
+            children = (
+                <LogConsole
+                    t={t}
+                    router={router}
+                    location={location}
+                    hasMore={hasMore}
+                    scrollToBottom={scrollToBottom}
+                    logArray={logArray}
+                    currentLogUrl={currentLogUrl}
+                    key={step.logUrl}
+                    prefix={`step-${step.id}-`}
+                />
+            );
         } else if (step.isInputStep) {
             children = <InputStep step={step} key="step" classicInputUrl={classicInputUrl} />;
         } else if (!logArray && step.hasLogs) {
             children = <span key={'span'}>&nbsp;</span>;
         }
         const getLogForNode = () => {
             if (this.pager.log === undefined || (this.pager.log && !this.pager.log.data)) {
-                const cfg = { url: step.logUrl };
+                const cfg = { url: step.logUrl, followAlong: this.props.augmenter.karaoke };
```

Comment:
... wheeeeeeee karaoke!

Reply:
Yes, the other changes I fixed for readability while trying to understand the issue. Figured no reason to throw them away and have to do it again next week.